src/include/daos_pool.h

```diff
@@ -49,9 +49,12 @@ typedef struct {
 	int			foo;
 } daos_target_perf_t;
 
-enum {
+
+/** Storage tier names */
+enum daos_media_type_t {
 	DAOS_MEDIA_SCM	= 0,
 	DAOS_MEDIA_NVME,
+	DAOS_MEDIA_NVME_PERF,
```

Comment:
Not sure why we need this. Could you please elaborate? It does not seem to be used anywhere in this patch. Please remove it for now.